.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,3 @@
-<!--
-Thank you for sending a PR. We appreciate you spending time to help improve the Aptos Core project.
+### Description
```

Comment:
@geekflyer, if I'm being picky, I'd like the template to have:
1. The header  `<!-- .... -->`. I think it's good to have this context and intro 😄.
2. The describing text for both sections, e.g.: `(Describe the proposed change here and the motivation for the change.)` and `(Share your test plan here. Please provide us with clear details for verifying that your changes work.)`

Having these here forces the author to think about them and allows reviewers to reject sloppy work.

Reply:
@JoshLind 

I honestly would prefer to have it really minimalist and only add what's relevant context required for 90+% of PRs.
The reasons for that are:
- remove noise from PRs where people forget to remove a section or just enter some dummy placeholder text (just a recent example: https://github.com/aptos-labs/aptos-core/pull/1316#issuecomment-1150445445 )
- reduce the number of keystrokes/copy/delete/select operations an author has to do to create a PR. I really like it to be intuitive such that when creating a PR I can enter a description right away.
- people are busy. The more noise/boilerplate there is a PR template, the less likely they're going to actually reading/thinking about the _important_ instructions in a PR template (like the fact that somewhere at the bottom we request a test plan).

With that in mind, I think:
1.  
> Thank you for sending a PR. We appreciate you spending time to help improve the Aptos Core project.

is pretty useless for most PRs (especially our own). I think it would be much more genuine and meaningful if we just say thank you in the PR review comment (which we do). If we still want to have have some kind of "automated thank you" we can add a bot or add it to to the contributing.md but I don't think it belongs into the PR template.

2. The text describing both sections
> Describe the proposed change here and the motivation for the change.
I think that's pretty obvious. Any good PR should describe the proposed change and motivation. Unfortunately I haven't seen an indication that having this placeholder text there actually _forces_ and author to do so or even increases the odds at having a PR good with a _good_ description. The only thing that truly impacts PR description quality is the review culture + reviewer stringency.

3.
> Share your test plan here. Please provide us with clear details for verifying that your changes work.
The `Share your test plan here. ` is pretty obvious from the Heading title, so we can remove that.
I think I'm fine keeping the `Please provide us with clear details for verifying that your changes work` just to make it a bit more explicit that we care about this. I'd probably wrap it in a "<!-- .... -->"  which make it hidden once submitted.

With that in mind the final PR template would look like this:
```
### Description

### Test Plan
<!-- Please provide us with clear details for verifying that your changes work. -->
```

Is that something agreeable? 😺 

Also btw, we can add more stuff back over time if we feel the need, but I'd like to start from a minimalist template and only add things as truly required.

Reply:
@geekflyer, I still have mixed feelings. 😄 In my mind, this description is for PR authors who are not part of the Aptos team, so I'm less worried about our PRs not following these instructions to the letter. However, I'll concede that having no CLA is a win. If I frequently find myself having to ask developers to add a description when I'm reviewing, I'll update the template. Unblocking this for now!

Reply:
> Unblock for no CLA! 🥳

ok cool, let's give it a shot and if we see that the PR quality deteriorates  we can add some more explicit instructions back to the template :)